cli/src/main/java/org/opentosca/toscana/cli/restclient/model/PlatformsResponse.java

```diff
@@ -0,0 +1,17 @@
+package org.opentosca.toscana.cli.restclient.model;
+
+public class PlatformsResponse {
+
+	private Platforms _embedded;
```

Comment:
Please adhere to Java naming conventions.

Reply:
The reason why this is here is the deserializer. that uses field names to Map the json object onto a java object. But therre are annotations to define the specific field name in the json object.

I dont know what you are using for this purpose (has to be definied when creating a retrofit instance)
With Gson you can add:
`@SerializedName("_embedded")`
With Jackson (The one spring uses) you can use:
`@JsonProperty("_embedded")`